python/lsst/imgserv/locateImage.py

```diff
@@ -82,11 +82,20 @@ def getImageFull(self, ra, dec):
         Returns None if no image is found.
         This function assumes the entire image is valid. (no overscan, etc.)
         '''
+        img, metadata = self.getImageFullWithMetadata(ra, dec)
+        return img
+
+    def getImageFullWithMetadata(self, ra, dec):
+        '''Return an image containing ra and dec with corresponding metadata.
+        Returns None if no image is found.
+        This function assumes the entire image is valid. (no overscan, etc.)
+        '''
         # The SQL UDF scisql_s2PtInBox requires a box, not a point.
         # 5 arcseconds is a small arbitrary box that seems to work.
```

Comment:
I'm not sure what is my attitude to magic numbers is. I know that we do not like magic, but I have no idea what is the correct way to avoid these numbers here.